api/api.go

```diff
@@ -240,3 +244,34 @@ func getAnnouncements(w http.ResponseWriter, r *http.Request) {
 	w.Write(b)
 
 }
+
+func getMembers(w http.ResponseWriter, r *http.Request) {
+	query := r.URL.Query()
+	limit := viper.GetInt("api.event_query_limit")
+	servers := viper.Get("discord.servers").(*config.Servers)
+	queryAmount, exists := query["q"]
+	if !exists || len(query) == 0 {
+		http.Error(w, "Please add the parameter 'q'", 403)
+		return
+	}
+	amount, err := strconv.Atoi(queryAmount[0])
+	if err != nil {
+		http.Error(w, "Please provide an int as 'q's value", 403)
+		return
+	}
+	if amount > limit {
+		http.Error(w, "Query amount exceeds the query limit", 403)
```

Comment:
And here

Reply:
status 400 for bad request 403 is access denied

Reply:
500 bruh moment